docs/source/public/overview.rst

```diff
@@ -278,6 +278,50 @@ The directory should be removed and the job should be resubmitted (depending on
 
 2. The extract results script mentioned :ref:`here <inspecting results>` and the cleanup script mentioned :ref:`here <post-simulation cleanup>` both include output of which folder(s) and ``json`` file(s) contain errors and can be removed prior to starting new jobs.
 
+.. _fix_networks:
+
+Fixing broken networks
+----------------------
+
+If removing a reproducibly failing edge leads to a disconnected graph, new edges need to be added in order to fix the broken network.
+The fixing of disconnected networks is carried out using the script provided under
+`utils/fix_networks.py <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/fix_networks.py>`_.
+
+The script finds new edges to repair the disconnected network and produces the inputs necessary to run those additional edges.
+
+Note that the script will throw an error if at least one repeat completed successfully. In that case (non-reproducible failure) we recommend re-running the failed jobs (see :ref:`Handling failed edges <failed_edges>`
+
+Here an example of how to run the script:
+
+.. code-block:: bash
+
+   python fix_networks.py --input_alchem_network_file alchemicalNetwork/alchemical_network.json --result_files results_*/*json --output_extra_transformations new_edges
+
+The script takes as inputs the ``AlchemicalNetwork`` from the original setup, the result ``.json`` files, and the folder name for storing the outputs.
```

Comment:
Where should they expect to find this if they use the default instructions we gave them?